### ISSUE TITLE
Set a limit on the number of home page billboards

### ISSUE THREAD

**Issue by knice** [ucsc/wcms-design-templates#412]
We can discuss this after launch, but it should be discussed. My data from the homepage shows that the 3rd or 4th billboard rarely gets seen or clicked on. Having 8-10 billboards costs mobile users $$$ to download all of those images they will never see or click on.

**Comment by teresasilva71** [comment]
I totally agree! Nobody sits on a home page and waits for the billboards play.

**Comment by knice** [comment]
I'm not sure why we never set a limit when we first rolled out WCMS, but it seems like something worth discussing (after launch).
